torchgeo/datasets/utils.py

```diff
@@ -266,6 +266,41 @@ def download_radiant_mlhub_collection(
     collection.download(output_dir=download_root, api_key=api_key)
 
 
+def download_azure_container(
+    root: str, name_starts_with: str, *args: Any, **kwargs: Any
+) -> None:
+    """Download a container from Azure blob storage.
+
```

Comment:
I would just add some examples of how to actually instantiate ContainerClient here

Reply:
Would it help if we made `account_url` and `container_name` explicit parameters instead of putting them in args/kwargs? They are required by ContainerClient, but I didn't explicitly list them to keep the code as simple as possible. Either way, this function currently doesn't show up in the docs, as it's only intended for use in TorchGeo, not use in other programs.